website/docs/r/log_analytics_workspace.html.markdown

```diff
@@ -45,6 +45,10 @@ The following arguments are supported:
 
 * `daily_quota_gb` - (Optional) The workspace daily quota for ingestion in GB.  Defaults to -1 (unlimited).
 
+* `enable_ingestion_over_public_dns` - (Optional) Should the Log Analytics Workflow support ingestion over the Public Internet? Defaults to `true`.
+
+* `enable_query_over_public_dns` - (Optional) Should the Log Analytics Workflow support querying over the Public Internet? Defaults to `true`.
```

Comment:
then this would be
```suggestion
* `internet_query_enabled` - (Optional) Should the Log Analytics Workflow support querying over the Public Internet? Defaults to `true`.
```